src/ucp/core/ucp_rkey.h

```diff
@@ -128,6 +126,7 @@ typedef struct ucp_rkey {
             ucp_rkey_proto_index_t    rma_proto_index; /* Protocol for RMAs */
             uct_rkey_t                rma_rkey;        /* Key to use for RMAs */
             uct_rkey_t                amo_rkey;        /* Key to use for AMOs */
+            ssize_t                   max_put_short;   /* Cached value of max_put_short */
```

Comment:
can we make this "int16_t" ad move it after "rma_proto_index"
assuming all fields in the cache except uct_rkey_t are 1b it should fit into the hole